lib/DDG/Spice/Upc.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Upc;
+
+use DDG::Spice;
+
+attribution
+    github => "iambibhas",
+    twitter => "bibhasdn",
+    web => "https://bibhas.in";
+
+triggers any => "upc", "upc code";
+
+spice to => 'http://api.upcdatabase.org/json/{{ENV{DDG_SPICE_UPC_APIKEY}}}/$1';
+spice wrap_jsonp_callback => 1;
+spice is_cached => 1;
+spice proxy_cache_valid => "1d";
```

Comment:
Should specify the HTTP response as well because we only want to cache `200` responses only https://duck.co/duckduckhack/spice_advanced_backend#caching-api-responses